docs: add instructions to update corepack (#637)
nodejs/corepack: README.md

</details>

<details><summary>Update Corepack using npm</summary>

To install the latest version of Corepack, use:

```shell
npm install -g corepack@latest
```

> If you are a Windows user and you [downloaded and installed Node.js](https://nodejs.org/en/download) using a Windows Installer `.msi` package then you need to disable the "corepack manager" feature of the Node.js `.msi` package by selecting "Entire feature will be unavailable" before attempting to install a different version of Corepack using npm.
> You can select the Modify option of the Node.js app settings to access the Windows Installer feature selection.
> (See [Repair apps and programs in Windows](https://support.microsoft.com/en-us/windows/repair-apps-and-programs-in-windows-e90eefe4-d0a2-7c1b-dd59-949a9030f317) for instructions on accessing the Windows apps page to modify settings.)

Reviewer: We should be using conditional, even though it is the case today that this is needed, it might not hold true in the future.

```suggestion
If Corepack was installed on your system using a Node.js Windows Installer
`.msi` package then you might need to remove it before attempting to install a
different version of Corepack using npm. You can select the Modify option of the
Node.js app settings to access the Windows Installer feature selection, and on
the "corepack manager" feature of the Node.js `.msi` package by selecting
"Entire feature will be unavailable". See
[Repair apps and programs in Windows](https://support.microsoft.com/en-us/windows/repair-apps-and-programs-in-windows-e90eefe4-d0a2-7c1b-dd59-949a9030f317)
for instructions on accessing the Windows apps page to modify settings.
```
